dist/src/main/config/zeebe.cfg.toml

```diff
@@ -30,6 +30,41 @@
 
 # ----------------------------------------------------
 
+
+[gateway]
+# Enable the embedded gateway to start on broker startup
+#
+# This setting can also be overridden using the environment variable ZEEBE_EMBED_GATEWAY.
+# enable = true
+
+[gateway.network]
+# Sets the host the embedded gateway binds to
+#
```

Comment:
Maybe for the user is also interested in the order of precedence. What do you think?

Reply:
I mean this:
```
    // network host precedence from higher to lower:
    // 1. ENV_GATEWAY_HOST
    // 2. specified in gateway toml section
    // 3. ENV_HOST
    // 4. specified in broker network toml section
```